egs/librimix/ConvTasNet/run.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+set -e  # Exit on error
+
+# If you haven't generated LibriMix start from stage 0
+
+# Main storage directory. You'll need disk space to store LibriSpeech, WHAM and LibriMix this is about 500 Gb
+storage_dir=
+
+
+# After running the recipe a first time, you can run it from stage 3 directly to train new models.
+
+# Path to the python you'll use for the experiment. Defaults to the current python
+# You can run ./utils/prepare_python_env.sh to create a suitable python environment, paste the output here.
+#python_path=${storage_dir}/asteroid_conda/miniconda3/bin/python
+python_path=python
+
+# All the parameters
+# General
+tag=""  # Controls the directory name associated to the experiment
+
+# Network config
+n_blocks=8
+n_repeats=3
+mask_act=relu
+# Training config
+epochs=200
+batch_size=24
+num_workers=4
+half_lr=yes
+early_stop=yes
+# Optim config
+optimizer=adam
+lr=0.001
+weight_decay=0.
+# Data config
+train_dir=data/wav8k/min/train-360
+valid_dir=data/wav8k/min/dev
+test_dir=data/wav8k/min/test
+sample_rate=8000
+n_src=2
+segment=3
+task=sep_clean
+
+
+stage=0  # Controls from which stage to start
+
+. utils/parse_options.sh
+
+# Example usage
+# ./run.sh --stage 3 --tag my_tag --task sep_noisy --id 0,1
+
+if [[ $stage -le  0 ]]; then
+	echo "Stage 0: Generating Librimix dataset"
+  . local/prepare_data.sh --storage_dir $storage_dir --n_src $n_src
+fi
+
+# Generate a random ID for the run if no tag is specified
+uuid=$($python_path -c 'import uuid, sys; print(str(uuid.uuid4())[:8])')
+if [[ -z ${tag} ]]; then
+	tag=${uuid}
+fi
+
+exp_dir=exp/train_convtasnet_${tag}
+mkdir -p $exp_dir && echo $uuid >> $exp_dir/run_uuid.txt
+echo "Results from the following experiment will be stored in $exp_dir"
+
+
+if [[ $stage -le 1 ]]; then
+  echo "Stage 1: Training"
+  mkdir -p logs
+  $python_path train.py --exp_dir $exp_dir \
+  --n_blocks $n_blocks \
+  --n_repeats $n_repeats \
+  --mask_act $mask_act \
+  --epochs $epochs \
+  --batch_size $batch_size \
+  --num_workers $num_workers \
+  --half_lr $half_lr \
+  --early_stop $early_stop \
+  --optimizer $optimizer \
+  --lr $lr \
+  --weight_decay $weight_decay \
+  --train_dir $train_dir \
+  --valid_dir $valid_dir \
+  --sample_rate $sample_rate \
+  --n_src $n_src \
+  --segment $segment
+fi
+
+out_dir=librimix
```

Comment:
This should probably not be here but rather before stage 0, where things are read by the user. 
Or even before parse_options so that it can be specified from command line.